controls/ssh_spec.rb

```diff
@@ -40,6 +39,16 @@
     it { should_not be_writable.by('group') }
     it { should_not be_writable.by('other') }
   end
+  case os[:family]
```

Comment:
@techraf thank you! 

> and it would show up here, right?

right, this `fixup` thing was only a practice you see sometimes (and its useful for big PRs usually), I wanted only to mention it as an answer to your question:)

amend is totally ok (its the same like fixup commit and squash in one step in the single-commit situation)